catalog/app/utils/workflows.spec.js

```diff
@@ -36,14 +32,9 @@ describe('utils/workflows', () => {
         version: "1"
         workflows: []
       `
-      const config = workflows.parse(data)
-
+      const parser = () => workflows.parse(data)
```

Comment:
since this function isnt reused, i'd rather make it inline: `expect(() => workflows.parse(data)).toThrow`